mathesar_ui/src/component-library/dropdown/Dropdown.scss

```diff
@@ -1,5 +1,8 @@
 button.dropdown.trigger {
   position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
```

Comment:
This styling will cause a visual regression in dropdowns that are sized bigger than it's contents.

Examples., 
| Before | In this PR | Notes |
|-----|----|----|
| <img width="698" alt="Screenshot 2023-11-27 at 7 11 44 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/da4b96ba-b0bd-49ff-b8ee-95348cd1dbc1"> | <img width="698" alt="Screenshot 2023-11-27 at 7 11 35 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/764d0100-62e3-4548-9975-817e94e02346"> | Notice the dropdowns in the type selection menu. This is in the import preview page after you upload a file |
| <img width="385" alt="Screenshot 2023-11-27 at 7 06 16 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/df127e07-2c40-4c54-9f5a-749f41c91156"> | <img width="385" alt="Screenshot 2023-11-27 at 7 06 27 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/49b538fb-d48b-4b34-88aa-f200bfc83046"> | Notice the table selection dropdown in the Data explorer |

Note: This is just two places that stood out, there might be other places in the app that have similar regressions due to this change.

If you'd like to go through with this change, I'd recommend going through the entire app (usages of Dropdown and components that compose on top of Dropdown) and making sure that you fix all regressions. My recommendation however, is to revert this change.

Reply:
Removed from global dropdown styles and added specifically to the menu-container.